src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf

```diff
@@ -82,19 +67,14 @@
         <target state="translated">Der {1}-Typparameter enthält die Einschränkung "unmanaged". "{1}" kann daher nicht als Einschränkung für "{0}" verwendet werden.</target>
         <note />
       </trans-unit>
-      <trans-unit id="ERR_ConstantPatternVsOpenType">
```

Comment:
Is it expected that these untranslated resources are removed?

Reply:
The delivered xlf were loclalized based on 4/1 version of xlf file.
meanwhile there were several updates of xlf.
Can you merge or re-generate xlf after check-in?

Reply:
I'm working on fixing things up now.